pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
Sorry, forgot to tell you that aliases are indeed not supported in the contrib repo (validation fails).